app/src/main/java/com/nextcloud/talk/utils/TextMatchers.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Nextcloud Talk application
- *
- * @author Mario Danic
- * @author Tim Krüger
- * @author Andy Scherzinger
- * Copyright (C) 2022 Andy Scherzinger <info@andy-scherzinger.de>
- * Copyright (C) 2021 Tim Krüger <t@timkrueger.me>
- * Copyright (C) 2017-2018 Mario Danic <mario@lovelyhq.com>
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
- * Partly based on https://github.com/kevalpatel2106/EmoticonGIFKeyboard/blob/master/emoticongifkeyboard/src/main/java/com/kevalpatel2106/emoticongifkeyboard/internal/emoticon/EmoticonUtils.java
- */
-
-package com.nextcloud.talk.utils;
-
-import com.vanniktech.emoji.EmojiInformation;
-import com.vanniktech.emoji.Emojis;
-
-import androidx.annotation.Nullable;
-
-public final class TextMatchers {
-
-    public static boolean isMessageWithSingleEmoticonOnly(@Nullable final String text) {
```

Comment:
this must not be deleted.
See comments in *TextMessageViewHolders.

Here must be checked if the text contains a single emoji.
I guess google will give some hints how to do it with regex

Reply:
Resolved in the recent commit using regex